classes/security/authorization/internal/SubmissionFileRevisionStageRequiredPolicy.inc.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file classes/security/authorization/internal/SubmissionFileRevisionStageRequiredPolicy.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionFileRevisionStageRequiredPolicy
+ * @ingroup security_authorization_internal
+ *
+ * @brief Submission file policy to ensure that we have a file at revision stage.
+ *
+ */
+
+import('lib.pkp.classes.security.authorization.internal.SubmissionFileBaseAccessPolicy');
+
+class SubmissionFileRevisionStageRequiredPolicy extends SubmissionFileBaseAccessPolicy {
```

Comment:
In case it's useful elsewhere, why not make this policy more general? Have its constructor take a stage ID, and then it can be used to check whether a submission file is in an arbitrary stage.